py4DSTEM/process/polar/polar_peaks.py

```diff
@@ -741,6 +758,15 @@ def plot_radial_peaks(
     if not label_y_axis:
         ax.tick_params(left=False, labelleft=False)
 
+    if v_lines is not None:
+        y_min, y_max = ax.get_ylim()
+
+        if np.isscalar(v_lines):
+            ax.vlines(v_lines, y_min, y_max, color="g")
+        else:
+            for a0 in range(len(v_lines)):
+                ax.vlines(v_lines[a0], y_min, y_max, color="g")
```

Comment:
I believe the if/else isn't necessary - `ax.vlines(lines,ymin,ymax)` can accept tuples or lists for `lines` - i.e. we can replace these 5 lines with just line 765